src/main/java/se/liu/ida/hefquin/engine/federation/access/impl/reqproc/GraphQLRequestProcessorImpl.java

```diff
@@ -26,14 +41,61 @@ public GraphQLRequestProcessorImpl() {
 	public JSONResponse performRequest( final GraphQLRequest req,
 	                                    final GraphQLEndpoint fm )
 	                                    		throws FederationAccessException {
-		// TODO this function needs to be implemented
-		//
-		// To get the GraphQL query that is to be executed you can do
-		// req.getGraphQLQuery() and to get the URL of the GraphQL
-		// endpoint to which the request needs to be posted you can
-		// do fm.getInterface().getURL()
-
-		return null;
-	}
 
+		Date startTime = new Date();
+		GraphQLQuery query = req.getGraphQLQuery();
+		String url = fm.getInterface().getURL();
+		url += query.getURL();
```

Comment:
No need for the `+=` operation. Also, 
```suggestion
		final String url = fm.getInterface().getURL() + query.getURL();
```
Also, please make sure that the final URL makes sense (for instance, I'm not sure that the interface URL will always end with a /).